AGENTS.md

```diff
@@ -0,0 +1,197 @@
+# AGENTS.md
+
+## Setup
+
+### Requirements
+
+- Node.js >=20.0.0
+- Python 3
+- Git (for development)
+
+### Environment
+
+- Docker (required for E2E tests via CHT Docker Helper)
+- Internet connection (for downloading dependencies and connecting to CHT instances)
+
+### Installation
+
+```bash
+# Install cht-conf globally
+npm install -g cht-conf
+
+# Install required Python dependency
+pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic
+
+# For local development
+git clone https://github.com/medic/cht-conf.git
+cd cht-conf
+npm install
+```
+
+### Docker Alternative
+
+```bash
+# Run using Docker (example: run tests)
+docker run -it --rm -v "$PWD":/workdir medicmobile/cht-app-ide npm test
+# Or run end-to-end tests
+docker run -it --rm -v "$PWD":/workdir medicmobile/cht-app-ide npm run test-e2e
+# For more available commands, see the documentation at https://github.com/medic/cht-conf#usage or run:
+docker run -it --rm -v "$PWD":/workdir medicmobile/cht-app-ide --help
+```
```

Comment:
This section is a bit confusing since the `cht-app-ide` docker image is for _using_ cht-conf, not _building/developing_ cht-conf.  I think we can just remove it.

```suggestion
```